model-optimizer/mo/main.py

```diff
@@ -24,7 +24,8 @@
 from mo.utils import import_extensions
 from mo.utils.cli_parser import get_placeholder_shapes, get_tuple_values, get_model_name, \
     get_common_cli_options, get_caffe_cli_options, get_tf_cli_options, get_mxnet_cli_options, get_kaldi_cli_options, \
-    get_onnx_cli_options, get_mean_scale_dictionary, parse_tuple_pairs, get_freeze_placeholder_values, get_meta_info
```

Comment:
Some changes in this file break python2 check in the MO:

> python2.7 mo.py
> Traceback (most recent call last):
>   File "mo.py", line 8, in <module>
>     from mo.main import subprocess_main  # pylint: disable=no-name-in-module
>   File "<PATH>/model-optimizer/mo/main.py", line 40
>     def replace_ext(name: str, old: str, new: str):
>                         ^
> SyntaxError: invalid syntax


You should remove functions annotations for some methods.

Reply:
To avoid a lot of changes in main.py I moved subprocess_main into separate file and changed it to be python2 compatible. So no python2 check works. See https://github.com/openvinotoolkit/openvino/pull/5504/commits/0e76833bd761759de125f9f02e8f33deb2dbc539